components/Card.js

```diff
@@ -21,13 +21,29 @@ export default function Card({
         </DeleteButton>
       )}
       <p>Date: {date}</p>
-      {weight && <p>Weight: {weight}</p>}
-      {height && <p>Height: {height}</p>}
-      {feastTime && (
-        <p>
-          Nurse time: {feastTime.substr(0, 2)}:{feastTime.substr(2, 2)}:
-          {feastTime.substr(4, 2)}
-        </p>
+      {weight?.[0] && (
```

Comment:
You don't need the optional chaining operator here. if the first entry is not available if will default to `undefined`. The best method here would be weightList.length === 0 && ...

Reply:
thanks for the hint, I changed it to ```array.lenght !== 0```  and implemented it

Reply:
It seems I really do need the optional chaining operator here, as on the attributes pages (weights for now) i dont pass all of the arrays down

Reply:
ok, now I'm handing it an empty array